chrome/content/zotero/xpcom/recognizePDF.js

```diff
@@ -1,39 +1,39 @@
 /*
     ***** BEGIN LICENSE BLOCK *****
-    
```

Comment:
Just, curious, is this just to avoid having a bunch of changed lines in the repo, or is there a specific reason you want the spaces on the empty lines?

Reply:
Both. Patches shouldn't have irrelevant changed lines, and Zotero style calls for (or at least allows for) indented newlines, which make editing less annoying in some editors.

You can revert these by doing `git reset HEAD^` and using `git checkout -p` to undo these changes, and then commit everything else and force-push.